src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -351,13 +348,13 @@ case class MultiPredictionBaggedResult(
     val arg = IJMat2 + JMat2
     Async.canStop()
 
-    /* Avoid division in the loop */
-    val inverseSize = 1.0 / modelPredictions.head.size
+    /* Avoid division in the loop, calculate (n - 1) / (n * B^2) */
+    val prefactor = 1.0 / Math.pow(modelPredictions.head.size, 2.0) * (Nib.size - 1) / Nib.size
 
     modelPredictions.indices.map { i =>
       Async.canStop()
       /* Compute the first order bias correction for the variance estimators */
-      val correction = Math.pow(inverseSize * norm(predMat(::, i) - meanPrediction(i)), 2)
+      val correction = prefactor * Math.pow(norm(predMat(::, i) - meanPrediction(i)), 2)
```

Comment:
Insofar as avoiding division in the loop is a relevant optimization here, s/`Math.pow(norm(predMat(::, i) - meanPrediction(i)), 2)`/`norm(predMat(::, i) - meanPrediction(i)) * norm(predMat(::, i) - meanPrediction(i))`/ (where `norm(predMat(::, i) - meanPrediction(i))` is actually factored out into its own `val` preceding this line) might also be worthwhile. Unless `Math.pow` includes that optimization already. (But of course that's beyond the point of this PR.)

Reply:
Looking at the docstring for Java's `pow` method, which lists several special cases, it doesn't look like there's a special case for "the second argument is an integer" (there is a special case for "both arguments are integers). So there might be room for improvement in several places by writing out `x * x` instead of `math.pow(x, 2)`